cmd/crossplane/main.go

```diff
@@ -19,11 +19,9 @@ package main
 import (
 	"os"
 	"path/filepath"
-
```

Comment:
Looks like these newlines were inadvertently removed and are causing errors.

Reply:
This is a demo commit to show the error in `lint` doesn't leak somewhere else in the CI. I will revert them after review is done.